x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupJobIdentifierUtils.java

```diff
@@ -97,11 +98,13 @@ private static void checkDateHisto(DateHistogramAggregationBuilder source, List<
                     if (agg.get(RollupField.AGG).equals(DateHistogramAggregationBuilder.NAME)) {
                         DateHistogramInterval interval = new DateHistogramInterval((String)agg.get(RollupField.INTERVAL));
 
-                        String thisTimezone  = (String)agg.get(DateHistogramGroupConfig.TIME_ZONE);
-                        String sourceTimeZone = source.timeZone() == null ? DateTimeZone.UTC.toString() : source.timeZone().toString();
+                        ZoneId thisTimezone = ZoneId.of(((String)agg.get(DateHistogramGroupConfig.TIME_ZONE)), ZoneId.SHORT_IDS);
+                        ZoneId sourceTimeZone = source.timeZone() == null
+                            ? ZoneId.of(DateHistogramGroupConfig.DEFAULT_TIMEZONE)
```

Comment:
Using `ZoneId.of("UTC")` (as this effectively does) causes printing using formats with this ZoneId to be messed up. They will get a `[UTC]` appended to the end of the time, like `1970-01-02T10:17:36.789Z[UTC]`. Instead, use `ZoneOffset.UTC`.

Reply:
Urgh, yeah not what was intended.  Fixing, thanks!